src/components/languageselector/LanguageSelector.tsx

```diff
@@ -60,7 +60,7 @@ export class LanguageSelector extends React.Component<LanguageSelectorProps> {
 
     return (
       <div className={containerClasses}>
-        <div style={{ position: 'fixed', right: '2em', marginTop: '2em', zIndex: 1 }}>
+        <div className={styles.Languages}>
```

Comment:
```suggestion
        <div className={styles.languages}>
```

The className convention in floweditor is camel case